magma/lib/magma/server.rb

```diff
@@ -25,6 +26,9 @@ def initialize
 
     post '/update_model', action: 'update_model#action', auth: { user: { is_admin?: :project_name } }
 
+    get '/flags/:project_name/', action: 'flags#get', auth: { user: { can_view?: :project_name } }
+    post '/flags/:project_name/', action: 'flags#set', auth: { user: { is_admin?: :project_name } }
+
```

Comment:
not sure what if any effect of trailing / is here, but usually routes don't have this